doc/developer-guide.rst

```diff
@@ -58,9 +59,11 @@ Usage
 The nominal usage patterns are covered in the :ref:`lsst.ts.DSM.user_guide`
 
 If the DSM is run in real mode for testing with the DSM UI, the telemetry directory
-that the UI uses must be mounted into the container. If one uses the repository
-location directory to create a directory (say ``telemetry``), this is handled by the 
-``docker run`` command shown above. However, the DSM CSC expects the telemetry directory to be in ``/home/saluser/telemetry``. This can be overridden by using the ``DSM_TELEMETRY_DIR`` environment variable to set the new path. It should be:
+that the UI uses must be mounted into the container.
+If one uses the repository location directory to create a directory (say ``telemetry``), this is handled by the ``docker run`` command shown above.
+However, the DSM CSC expects the telemetry directory to be in ``/home/saluser/telemetry``.
+This can be overridden by using the ``DSM_TELEMETRY_DIR`` environment variable to set the new path.
+It should be:
```

Comment:
I'm a bit confused by "It should be:". This suggests the CSC will fail if it is set to some other directory, yet the suggested path doesn't even match the default. I suggest clarifying why it should have this value, or possibly omitting this bit.

below you say "and done in the development container before running the startup script". Please consider "set" instead of "done" (I'm not sure what "done" means in this case).